### PR TITLE
User update

### PR DESCRIPTION
**Pull request by bajajerk**

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -33,6 +37,14 @@ def reset_password
     end
   end
 
+  def update
+    if @current_user.update(user_params)
+      redirect_to '/', notice: 'Your profile was successfully updated.'
```
Comment (abhishek77in) [inline app/controllers/users_controller.rb]: Redirect user to `users#show`

File: app/views/patient_registration/users/show.haml [hunk 2]
```diff
@@ -4,3 +4,5 @@
   = link_to 'Change my password',
       edit_patient_registration_user_path(current_user),
       class: 'button secondary'
+  = link_to 'Update Details', edit_user_path(current_user),
+      class: 'button secondary'
```
Comment (abhishek77in) [inline app/views/patient_registration/users/show.haml]: Always add a new line in the end of file.

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (bajajerk) [reply app/controllers/users_controller.rb]: There's no method show on user controller in the codebase

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -33,6 +37,14 @@ def reset_password
     end
   end
 
+  def update
+    if @current_user.update(user_params)
+      redirect_to patient_registration_user_path(current_user), notice: 'Your profile was successfully updated.'
```
Comment (bajajerk) [inline app/controllers/users_controller.rb]: Is this fine or should I make a generalised show method in user controller

Comment (abhishek77in) [reply app/controllers/users_controller.rb]: Show does exist - `app/views/patient_registration/users/show.haml`

Comment (bajajerk) [reply app/controllers/users_controller.rb]: I have redirected to this .

Comment (abhishek77in) [reply app/controllers/users_controller.rb]: @bajajerk Please use `PatientRegistration::UsersController` for this story. And not use `UsersController`.

**Pull request closed without merging**